### PR TITLE
Create a team in postman for ks-devops

### PR DESCRIPTION
**Pull request by AbdelouahabMbarki**
**issue:** #48
**description :** include ks-devops postman team link in the readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +63,8 @@ curl -H "Authorization: Bearer xxxx" \
 
 > Please get a token from Kubernetes cluster, and replace `xxxx` with it.
 
+If  you want to see ks-devops postman api collection , please visit **[ks-devops postman](https://www.postman.com/ks-devops/workspace/kubesphere-devops)**
```
Comment (LinuxSuRen) [inline README.md:66]: There is an extra space here.

`api` -> `API`

**Review by LinuxSuRen — CHANGES_REQUESTED** [review]
Thanks for taking the initiative. I found a few issues. Please take a look at them.

By the way, usually if a PR can close an issue, then the following comment is recommend.

`fix #48`

I'm sure I know `48` is the number of the related issue. You can see the similar tips from the corresponding issue below:

image

File: README.md (hunk 1 above)
Comment (AbdelouahabMbarki) [reply README.md]: thank you , it's  fixed  now

**Comment by AbdelouahabMbarki** [comment]
fix #48

**Comment by AbdelouahabMbarki** [comment]
fix #48

**Review by LinuxSuRen — COMMENTED** [review]
/lgtm

/label tide/merge-method-squash

**Pull request merged**